core/src/main/java/com/microsoft/applicationinsights/internal/util/PropertyHelper.java

```diff
@@ -28,8 +30,9 @@
 /**
  * Helper class for reading data from a properties file found on the class path.
  */
-public class PropertyHelper
-{
+public final class PropertyHelper {
+    public final static String SDK_VERSION_FILE_NAME = "sdk-version.properties";
```

Comment:
This is a generic helper class while "sdk-version.properties" is a file designated for only for specific purpose (getting sdk-version numbers) so I wouldn't mix these two. A better design would be to define another class for extracting these properties and then it can be used both by the SdkVersionContextInitializer and your code for placing the native artifacts